lib/reporter.js

```diff
@@ -12,7 +12,14 @@ var generateReport = function (options) {
 
     var featureOutput = jsonFile.readFileSync(options.jsonFile);
     var packageJsonPath = searchFileUp('package.json');
-    var packageJson = packageJsonPath && jsonFile.readFileSync(packageJsonPath, 'utf8');
+    var packageJson = {}
+    try {
+        packageJson = packageJsonPath && jsonFile.readFileSync(packageJsonPath, 'utf8');
+    } catch (err) {
+        console.warn('No package.json file found in: ' + packageJsonPath + ', using default name and version.');
+        packageJson.name = 'default';
+        packageJson.version = '0.0.0';
```

Comment:
IMO - let's keep these key-pairs empty and log the warning. Will update before publishing. Thanks for pushing the commits.